src/AlgoliaEngine.php

```diff
@@ -1,35 +1,34 @@
 <?php
 
-namespace Algolia\SearchBundle\Engine;
+namespace Algolia\SearchBundle;
 
-use Algolia\SearchBundle\SearchableEntityInterface;
-use AlgoliaSearch\Client;
+use Algolia\AlgoliaSearch\SearchClient as Client;
 
-class AlgoliaEngine implements EngineInterface
+class AlgoliaEngine
```

Comment:
This class should be final, don't see a reason for people overwrite in it.

Reply:
I'd say there could be use cases. Let's discuss this!